relay-general/src/store/clock_drift.rs

```diff
@@ -115,14 +123,15 @@ impl Processor for ClockDriftProcessor {
         _meta: &mut Meta,
         state: &ProcessingState<'_>,
     ) -> ProcessingResult {
-        event.process_child_values(self, state)?;
-
         if let Some(correction) = self.correction {
+            event.process_child_values(self, state)?;
+
             let timestamp_meta = event.timestamp.meta_mut();
             timestamp_meta.add_error(Error::with(ErrorKind::InvalidData, |e| {
-                let reason = format!(
-                    "clock drift: all timestamps adjusted by {}",
-                    HumanDuration(correction.drift)
+                let mut reason = format!(
+                    "clock drift: all timestamps adjusted by {}. Reason: {}",
```

Comment:
What if you format the reason in the beginning, since this is rendered as reason in the UI?

```suggestion
                    "{}: all timestamps adjusted by {}",
```